module/eth/gas_tip_cap_test.go

```diff
@@ -0,0 +1,22 @@
+package eth_test
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/lmittmann/w3"
+	"github.com/lmittmann/w3/module/eth"
+	"github.com/lmittmann/w3/rpctest"
+)
+
+func TesGasTipCap(t *testing.T) {
```

Comment:
The test is never executed because of the missing `t` in `func Tes...`. Also please add the `eth_maxPriorityFeePerGas.golden`-file